semantic-reranker/src/main/java/org/opensearch/search/relevance/control/KendraSearchExtBuilder.java

```diff
@@ -17,29 +18,35 @@
 import org.opensearch.common.xcontent.XContentBuilder;
 import org.opensearch.common.xcontent.XContentParser;
 import org.opensearch.search.SearchExtBuilder;
+import org.opensearch.search.relevance.constants.Constants;
 
 public class KendraSearchExtBuilder extends SearchExtBuilder {
-  public static final String NAME = "kendra_intelligent_ranking";
+  public static final String NAME = Constants.PLUGIN_NAME;
 
   private static final ObjectParser<KendraSearchExtBuilder, Void> PARSER;
-  private static final ParseField RANKER_ENABLED = new ParseField("enabled");
+  private static final ParseField RANKER_ENABLED = new ParseField(Constants.ENABLED);
+  private static final ParseField BODY_FIELD = new ParseField(Constants.BODY_FIELD);
```

Comment:
I was chatting with some folks about the long-term strategy where we'll support multiple rerankers, and trying to make sure that users of this plugin will be able to send the same queries in the future.

The proposed spec is described in https://github.com/opensearch-project/search-relevance/issues/12.

Could we change the ext parameters to something more like:

```
{
  "ext": {
    "rankers": [
      {
        "name" : "kendra_intelligent_ranking",
        "properties" : {
          "body_field": [
            "field1",
            "field2"
          ]
        }
      }
    ]
   }
 }

Reply:
I want to propose a slightly modified structure
```
{
  "ext": {
    "relevancy_config": {
      "rankers": [
        {
          "name" : "kendra_intelligent_ranking",
          "enabled": true
          "properties" : {
            "body_field": [
              "field1",
              "field2"
            ]
          }
        }
      ]
    }
  }
}
```

1. We need an ExtBuilder NAME to identify the ext config specified by this plugin, under which the inputs will be nested (I’m using `relevancy_config` used as a placeholder). See corresponding [code][1] and [usage][2] from LTR as an example.
2. We need to have a flag to enable/disable rankers if we are to support index-level and request-level configuration. For example:
```
PUT my-index-000001/_settings
{
  "index" : {
    "kendra_intelligent_ranking" : {
        "enabled" : "true",
        "body_field": "text"
    }
  }
}
```
will enable the Kendra ranker for `my-index-000001`, but issuing a query without an `ext` entry like below, will override this and disable the Kendra ranker for this request, when in fact, it should be enabled.
```
GET my-index-000001/_search
{
  "query": {
    "match": {
      "text": {
        "query": "research"
      }
    }
  }
}
```

[1]:https://github.com/o19s/elasticsearch-learning-to-rank/blob/9f0612cf24fe5d700cc4ab6fa274c09d51841d91/src/main/java/com/o19s/es/ltr/logging/LoggingSearchExtBuilder.java#L38

[2]: https://elasticsearch-learning-to-rank.readthedocs.io/en/latest/logging-features.html#logging-values-for-a-live-feature-set